src/critics/local_goal_critic.cpp

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 @artofnothingness Alexey Budyakov, Samsung Research
+#include "mppic/critics/local_goal_critic.hpp"
```

Comment:
Needs to be in the critics XML with a description of the plugin so that its loadable by the system

Reply:
added